operations/image_management/Configure_a_Remote_Build_Node.md

```diff
@@ -688,3 +688,77 @@ Expected output will be something like:
     }
 ]
 ```
+
+### Checking the status of remote build nodes
+
+Before a job is set up to be launched on a remote build node, IMS checks that it is
+ready to run remote jobs. If a node is registered as a remote build node, but is not
+able to run remote jobs, that node will not have remote jobs scheduled on it. The
+status of the remote build nodes may be queried. The status object will have a field
+`ableToRunJobs` that contains a boolean indicating if the node is able to run jobs.
+If the node is not able to run jobs, the status object will document the issue with
+running jobs on that node.
+
+* NOTE: a value of '10000' for `numCurrentJobs` indicates that the actual number of
```

Comment:
10,000 seems weird and slightly unintuitive. I'd look at that and go, "Woah, busy node."

Reply:
Yeah, I struggled a little on this one but the main point is to draw attention to the node.

Reply:
You are right - changed it to '-1' to signify unknown.